### PR TITLE
node/tools: update dlv to v1.9.1

### PR DESCRIPTION
**Pull request by tbjump**

### CODE REVIEW

File: node/tools/go.mod [hunk 1]
```diff
@@ -1,28 +1,5 @@
 module github.com/certusone/wormhole/node/tools
 
-go 1.19
+go 1.16
```
Comment (bruce-riley) [inline node/tools/go.mod:3]: Why are we rolling back the version of go? If this is intentional, please add a comment.

Comment (hendrikhofstadt) [reply node/tools/go.mod:3]: This should not be necessary and fail with >1.19. Also why are all the indirects dropped?

**Review by hendrikhofstadt — CHANGES_REQUESTED** [review]
See inline comments

File: node/tools/go.mod (hunk 1 above)
Comment (tbjump) [reply node/tools/go.mod:3]: Idk. With go 1.19 I get a build error. 

How do I generate the indirects?

Comment (hendrikhofstadt) [reply node/tools/go.mod:3]: `go mod tidy`

**Review by hendrikhofstadt — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]

**Pull request merged**